src/test/java/uk/gov/hmcts/probate/schedule/FetchDraftCasesWithPaymentTaskTest.java

```diff
@@ -41,8 +43,7 @@ class FetchDraftCasesWithPaymentTaskTest {
 
     @BeforeEach
     void setUp() {
-        clock = Clock.systemDefaultZone();
-
+        clock = Clock.system(ZoneId.of("Europe/London"));
```

Comment:
If we are going to change this to use a different clock it might be preferable to use a fixed clock (and then adapt the tests such that they are expecting the fixed dates that will return
```suggestion
        clock = Clock.fixed( 
                Instant.parse("2025-10-02T03:00:00Z"),
                ZoneOffset.UTC);
```

this would mean that the `DATE_TODAY` value on line 41 would need to be fixed so as not to reference `LocalDate.now()` and instead would be `"2025-10-02"`